src/GitInfo/build/GitInfo.targets

```diff
@@ -50,6 +50,14 @@
 					 in a previous build in a GitInfo.cache for 
 					 performance reasons. 
 					 Defaults to empty value (no ignoring).
+           
+  $(GitTagRegex): Regular Experssion used with git describe to find the BaseTag
+           Defaults to * (all)
+           
+  $(GitBaseVersionExpr): Regular Experssion used with git describe to find the BaseTag

```

Comment:
The versions are semver2, I don't think we need to make the regex customizable

Reply:
If you're referring to making the GitBaseVersionExpr public instead of private, I do think you should allow user customization.  As written, the _GitBaseVersionExpr catches any tag with a semver2 version embedded.  Customizing it would allow us to be more specific about the format of the actual tag that we would consider as a "valid" version tag.